measure_extinction/extdata.py

```diff
@@ -423,15 +423,41 @@ def calc_AV(self, akav=0.112):
 
         # if no SpeX spectrum is available: compute A(V) from E(K-V)
         else:
-            dwaves = np.absolute(self.waves["BAND"] - 2.19 * u.micron)
-            sindxs = np.argsort(dwaves)
-            kindx = sindxs[0]
-            if dwaves[kindx] > 0.02 * u.micron:
-                warnings.warn("no K band measurement in E(lambda-V)", UserWarning)
+            kindx = np.where(self.waves["BAND"] == 2.19 * u.micron)[0][0]
```

Comment:
The previous code allowed some small variation in the K band central wavelength.  The idea here was that not all K bands were exactly the same (K vs Ks).   Is there a reason why this flexibility is not desired?  It was only allowing a variation of 0.02 micron.

Reply:
That is a good point. I thought the only purpose of the code was to find the K-band index. I did not realize the idea was to have some flexibility in the wavelength. However, the K-band wavelength has been hardcoded in `stardata.py` in the `get_poss_bands` function, so I don't see how there can be any variation in this wavelength as long as it's defined in that function.

Reply:
You are right, it isn't an issue now, but it could be in the future.  Just a thought.

Reply:
Ok, I'll add some more flexibility.